04LessonFour/in-class-project/index.html

```diff
@@ -3,8 +3,45 @@
 <head>
   <title>The Daily Planet</title>
   <link rel="stylesheet" href="css/style.css">
+
+  <div class="head">
+
+    <h1>Intel</h1>
+
+<style>
+  a {
+    color: white;
+  }
+</style>
+
+<nav>
+    <a href="link">i7 Processor</a>
+    <a href="link">Solid-State Drives</a>
+</nav>
+
+  </div>
+
 </head>
 <body>
 
+  <h2>Intel&reg; Core&trade;</h2>
+  <p>The 6th Generation Intel&reg; Core&trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds adn peak performance through even the toughest tasks and games.</p>
+
+  <div id="two">
+    <h2>Ready. Wake. Create.</h2>
+    <p>From zero power in a flash, the 6th Generation Intel&reg; Core&trade; processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever.</p>
+  </div>
+
+<!--Interal image of processor goes here-->
+<aside>
+  <img src="img/processor.jpg">
+</aside>
+
+<div id="three">
+    <h2>Intel Customer Support</h2>
+    <p>Find Support Topics by Product, Drivers &amp; Software, or ask the community <a href="link">read more</a>
+</div>
+

```

Comment:
I need your Checkpoint and the lesson 6 in-class project done asap! http://madz09.github.io/intro-workbook/06LessonSix/in-class-project/intel.html